@xen-orchestra/web-core/lib/components/UiChip.vue

```diff
@@ -0,0 +1,181 @@
+<template>
+  <span class="ui-chip typo p3-regular" :class="[color, { disabled }]">
+    <span class="content" @click="emit('edit')">
+      <slot name="icon">
+        <UiIcon v-if="icon" :icon class="icon" fixed-width />
+      </slot>
+      <slot />
+    </span>
+    <UiIcon :icon="faCircleXmark" class="remove" @click.stop="emit('remove')" />
```

Comment:
According to Design System, this should be a `ButtonIcon`

```suggestion
    <ButtonIcon
      :color="disabled ? false : color"
      :disabled
      :icon="faCircleXmark"
      size="small"
      @click.stop="emit('remove')"
    />
```

I passed `:color="disabled ? false : color"` because the remove icon takes on the color of the tag except when the tag is disabled.

This will let us handle its color manually in that case.

It needs a little adjustment on the `ButtonIcon` component:
```ts
  color?: Color | false
```